packages/manager/src/components/user/LoginPage.tsx

```diff
@@ -52,22 +77,47 @@ export default () => {
   return (
     <Page header={<Header />}>
       <PageSection variant={PageSectionVariants.light} isFilled>
-        <Bullseye>
-          <Card className={css(styles.card)}>
-            <CardHeader>
-              <Title headingLevel="h6" size="md">
-                Sign in with
-              </Title>
-            </CardHeader>
-            <CardBody>
-              <Button isBlock variant="primary" onClick={onClickLogin} className="spaship_btn">
-                Red Hat SSO
-              </Button>
-            </CardBody>
-          </Card>
-        </Bullseye>
+        <div className={css(styles.defcard)}>
+          <h5 className={css(styles.h5)}>
+            develop fast · <span className={css(styles.yellow)}>deploy faster</span>
+          </h5>
+          <div className={css(styles.definition)}>
+            SPAship is a open source platform for deploying, integrating, and managing single-page apps (SPAs).
+          </div>
+        </div>
+        <Card className={css(styles.card)}>
+          <CardHeader>
+            <Title headingLevel="h6" size="md">
+              Sign in with
+            </Title>
+          </CardHeader>
+          <CardBody>
+            <Button isBlock variant="primary" onClick={onClickLogin} className="spaship_btn">
+              Red Hat SSO
+            </Button>
+          </CardBody>
+        </Card>
+      </PageSection>
+      <PageSection
+        className={css(styles.footer)}
+        variant={PageSectionVariants.darker}
+        style={{ height: "6em" }}
```

Comment:
Why not move this into the top style definition ?
```suggestion

```

Reply:
Will do!